### DISCUSSION TITLE
Postgress password auth failed

### DISCUSSION THREAD

**Discussion by Tristannos** [serverpod/serverpod#682] [Q&A] [answered]
I wanted to test and start my serverpod server.
But when i try to run it, for some reason the password is incorrect, but when i go in docker the password is the same as in the passwords.yaml file.

And for some reason i cant upgrade or get server pod 1.0.1 or even 1.0.
This is the error i am getting in Android studio:

tristan@Tristans-MacBook-Pro rallyy_server % serverpod run
Starting Serverpod.

• Automatic generate and reload are enabled.

Spinning up serverpod generate (this can take a few seconds).
Waiting for Postgres on localhost:8090.
Waiting for Redis on localhost:8091.
Setup complete. Starting the server.

SERVERPOD version: 0.9.10 mode: development time: 2023-02-20 20:28:34.383504Z
2023-02-20 20:28:34.826300Z Failed to connect to database.
PostgreSQLSeverity.fatal 28P01: password authentication failed for user "postgres" 
===== asynchronous gap ===========================
package:serverpod_postgres_pool/postgres_pool.dart 318:18       PgPool.run.<fn>
package:retry/retry.dart 131:16                                 RetryOptions.retry
package:serverpod_postgres_pool/postgres_pool.dart 316:14       PgPool.run
package:serverpod/src/database/database_connection.dart 171:20  DatabaseConnection.find
package:serverpod/src/database/database_connection.dart 199:18  DatabaseConnection.findSingleRow
package:serverpod/src/database/database.dart 86:12              Database.findSingleRow
package:serverpod/src/server/serverpod.dart 312:13              Serverpod.start.<fn>
package:serverpod/src/server/serverpod.dart 301:5               Serverpod.start
package:rallyy_server/server.dart 30:3                          run

2023-02-20 20:28:36.221102Z Internal server error. Failed to load method lookup.
PostgreSQLSeverity.fatal 28P01: password authentication failed for user "postgres" 
===== asynchronous gap ===========================
package:serverpod_postgres_pool/postgres_pool.dart 318:18       PgPool.run.<fn>
package:retry/retry.dart 131:16                                 RetryOptions.retry
package:serverpod_postgres_pool/postgres_pool.dart 316:14       PgPool.run
package:serverpod/src/database/database_connection.dart 171:20  DatabaseConnection.find
package:serverpod/src/database/database_connection.dart 199:18  DatabaseConnection.findSingleRow
package:serverpod/src/database/database.dart 86:12              Database.findSingleRow
package:serverpod/src/server/method_lookup.dart 47:26           MethodLookup._attemptLoad
package:serverpod/src/server/method_lookup.dart 30:7            MethodLookup.load
package:serverpod/src/server/serverpod.dart 327:9               Serverpod.start.<fn>
package:serverpod/src/server/serverpod.dart 301:5               Serverpo

And this is the error from docker in postgresSQL: 
PostgreSQL Database directory appears to contain a database; Skipping initialization
2023-02-20 20:17:10.310 UTC [1] LOG:  starting PostgreSQL 14.1 (Debian 14.1-1.pgdg110+1) on aarch64-unknown-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
2023-02-20 20:17:10.310 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2023-02-20 20:17:10.310 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2023-02-20 20:17:10.311 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2023-02-20 20:17:10.314 UTC [26] LOG:  database system was shut down at 2023-02-20 20:14:59 UTC
2023-02-20 20:17:10.319 UTC [1] LOG:  database system is ready to accept connections
2023-02-20 20:17:35.923 UTC [35] FATAL:  password authentication failed for user "postgres"
2023-02-20 20:17:35.923 UTC [35] DETAIL:  Connection matched pg_hba.conf line 100: "host all all all scram-sha-256"
2023-02-20 20:17:35.997 UTC [38] FATAL:  password authentication failed for user "postgres"
2023-02-20 20:17:35.997 UTC [38] DETAIL:  Connection matched pg_hba.conf line 100: "host all all all scram-sha-256"
2023-02-20 20:17:36.060 UTC [39] FATAL:  password authentication failed for user "postgres"
2023-02-20 20:17:36.060 UTC [39] DETAIL:  Connection matched pg_hba.conf line 100: "host all all all scram-sha-256"
2023-02-20 20:17:36.125 UTC [42] FATAL:  password authentication failed for user "postgres"

**Comment by Pingear** [comment, marked as answer]
I can't tell you about the upgrade, but about the wrong password, I recently encountered a similar situation - https://github.com/serverpod/serverpod/discussions/677
Did you by any chance create a project with the same name as the previously deleted project? If yes, try to remove the corresponding volume from docker volumes.

**Reply by Tristannos** [reply]
Thank you!!!

I never used docker before so I thought by removing the container that everything was removed...

Can somebody else help me with why i cant upgrade to 1.0 or 1.0.1?

**Reply by yahu1031** [reply]
any issue it says?

ideally, this is what you should do `dart pub global activate serverpod_cli` to install/upgrade.

**Reply by Tristannos** [reply]
No, it says:

tristan@Tristans-MacBook-Pro ~ % dart pub global activate serverpod_cli  
Resolving dependencies... 
+ _fe_analyzer_shared 38.0.0 (53.0.0 available)
+ analyzer 3.4.1 (5.5.0 available)
+ archive 3.3.6
+ args 2.3.1 (2.4.0 available)
+ async 2.9.0 (2.10.0 available)
+ clock 1.1.1
+ collection 1.17.0 (1.17.1 available)
+ colorize 3.0.0
+ convert 3.1.0 (3.1.1 available)
+ crypto 3.0.2
+ file 6.1.4
+ glob 2.1.1
+ http 0.13.5
+ http_parser 4.0.2
+ intl 0.17.0 (0.18.0 available)
+ js 0.6.5 (0.6.7 available)
+ meta 1.9.0
+ package_config 2.1.0
+ path 1.8.3
+ pointycastle 3.6.2
+ pub_semver 2.1.2 (2.1.3 available)
+ recase 4.1.0
+ serverpod_cli 0.9.10 (1.0.1 available)
+ serverpod_client 0.9.10 (1.0.1 available)
+ serverpod_serialization 0.9.10 (1.0.1 available)
+ serverpod_service_client 0.9.10 (1.0.1 available)
+ serverpod_shared 0.9.10 (1.0.1 available)
+ source_span 1.9.1
+ stream_channel 2.1.1
+ string_scanner 1.1.1 (1.2.0 available)
+ super_string 1.0.3
+ term_glyph 1.2.1
+ typed_data 1.3.1
+ uuid 3.0.7
+ watcher 1.0.2
+ web_socket_channel 2.3.0
+ yaml 3.1.1
Building package executables... (2.5s)
Built serverpod_cli:serverpod_cli.
Installed executable serverpod.
Activated serverpod_cli 0.9.10.

But it wont upgrade:
Maybe i have to upgrade my flutter/dart?

**Reply by yahu1031** [reply]
try this `dart pub global activate serverpod_cli 1.0.1`

**Reply by Tristannos** [reply]
Thankyou! I had to update my Dart.
